.python-version

```diff
@@ -0,0 +1,3 @@
+3.4.2
+2.7.9
+pypy-2.4.0
```

Comment:
This file is necessary because circleci uses `pyenv`, which is kind of broken. It defeats tox's `--skip-missing-interpreters` option by putting a `pypy` script on the `$PATH` which doesn't actually work unless you pin a specific version here. And once you specify a version of pypy, you have to also specify a precise version of regular python (so this list becomes another thing we have to keep up to date. It's immediately outdated since python 2.7.10 was just released today but is not yet available on circleci).